composer.json

```diff
@@ -15,10 +15,12 @@
         "psr-4": {"Matomo\\ReportingApi\\": "src/"}
     },
     "require": {
-        "guzzlehttp/guzzle": "~6.0"
+        "php": "^7.4|^8.0",
```

Comment:
7.3? Also we should use double pipe as OR operator as | is deprecated. See https://github.com/composer/composer/issues/6755

Reply:
I don't want to impose a change of PHP version to our users. They might still be on older versions of PHP for legacy reasons.